js/qz-tray.js

```diff
@@ -589,7 +587,25 @@ var qz = (function() {
             },
 
             /** Signing algorithm used on signatures */
-            signAlgorithm: "SHA1"
+            signAlgorithm: "SHA1",
+
+            needsSigned: function(callName) {
+                const undialoged = [
+                    "printers.getStatus",
```

Comment:
Just a note... I was curious why "printers.getStatus" wasn't signed and it's because you MUST start listening first, so order of operations mandates that.  I tried abusing this cross-site (starting listener on one website, but getting status from another) and it's handled properly, the second site won't allow getting the status from the former listener, so this is good despite how odd it looks in the listing.
https://github.com/qzind/tray/blob/48eda67666aa11bc29299d78d1568fe52e93cc0a/src/qz/ws/SocketMethod.java#L8